src/components/Reports/Reports.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import ReportsFilters from './ReportsFilters/ReportsFilters';
 import ReportsBox from './ReportsBox/ReportsBox';
 import ReportsPageRanking from './ReportsPageRanking/ReportsPageRanking';
-import ReportsTrafficSources from './ReportsTrafficSources/ReportsTrafficSources';
+import ReportsTrafficSourcesOld from './ReportsTrafficSources/ReportsTrafficSourcesOld';
```

Comment:
You couldn't add a flag into `ReportsTrafficSources` right? to avoid the duplication.

Reply:
if add flag, need to call another method and show another data too, if you saw the service the interface are differents @cbernat